### DISCUSSION TITLE
Mouse and keyboard stop working right after boot up

### DISCUSSION THREAD

**Discussion by proletentoaster** [fayaaz/mixxx-pi-gen#43] [Q&A]
Hey there,

when i boot up, and see the splash screen i am able to move the mouse curser, right after the splash screen is gone and Mixxx appears,  my mouse and keyboard stops working (tried different models, Logitech or HP, are recognized when i run lsusb).
Interesting point is that my touchscreen works (will be replaced by a bigger screen without touch, so i need to use mouse and keyboard).
Also the scroll knob of my controller works (i think it's not using MIDI as i also can control different applications with it).
Someone else got that problem or knows what to do? Not sure if this is a bug.

BR
proletentoaster

**Comment by fayaaz** [comment]
Could this possibly be a power draw problem? Your Pi might be drawing too much power.
I noticed this with early versions when testing but haven't had the problem since.
